docs/source/deployment/azure_functions.rst

```diff
@@ -0,0 +1,304 @@
+    * Sign in Azure CLI: https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli?view=azure-cli-latest
+
+* Docker is installed and running on your local machine
+
+    * Install instruction: https://docs.docker.com/get-docker/
+
+* Python 3.6 or above and required Pypi packages: `bentoml` and `scikit-learn`
+
+    * .. code-block:: bash
+
+            pip install bentoml scikit-learn
+
+
+Azure Functions deployment with BentoML
+---------------------------------------
+
+Run the example project from the :doc:`Quickstart guide <../quickstart>` to create an
+IrisClassifier BentoService saved bundle for the deployment:
+
+.. code-block:: bash
+
+    git clone git@github.com:bentoml/BentoML.git
+    python ./bentoml/guides/quick-start/main.py
+
+
+Use BentoML CLI tool to get the information of the IrisClassifier BentoService created
+above
+
+.. code-block:: bash
+
+    $ bentoml get IrisClassifier:latest
+
+    # Sample output
+    {
+      "name": "IrisClassifier",
+      "version": "20200618124555_310CE0",
+      "uri": {
+        "type": "LOCAL",
+        "uri": "/Users/bozhaoyu/bentoml/repository/IrisClassifier/20200618124555_310CE0"
+      },
+      "bentoServiceMetadata": {
+        "name": "IrisClassifier",
+        "version": "20200618124555_310CE0",
+        "createdAt": "2020-06-18T19:46:18.675900Z",
+        "env": {
+          "condaEnv": "name: bentoml-IrisClassifier\nchannels:\n- defaults\ndependencies:\n- python=3.7.3\n- pip\n",
+          "pipDependencies": "scikit-learn\npandas\nbentoml==0.8.1",
+          "pythonVersion": "3.7.3",
+          "dockerBaseImage": "bentoml/model-server:0.8.1"
+        },
+        "artifacts": [
+          {
+            "name": "model",
+            "artifactType": "SklearnModelArtifact"
+          }
+        ],
+        "apis": [
+          {
+            "name": "predict",
+            "inputType": "DataframeInput",
+            "docs": "BentoService API",
+            "inputConfig": {
+              "orient": "records",
+              "typ": "frame",
+              "is_batch_input": true,
+              "input_dtypes": null
+            },
+            "outputConfig": {
+              "cors": "*"
+            },
+            "outputType": "DefaultOutput",
+            "mbMaxLatency": 300,
+            "mbMaxBatchSize": 2000
+          }
+        ]
+      }
+    }
+
+
+======================================
+Deploy BentoService to Azure Functions
+======================================
+
+Deploy to Azure Functions requires a deployment name, BentoService in name:version
+format, and a valid Azure location.  You can find list of Azure locations by calling
+command `az account list-locations`.
+
+BentoML will create necessary Azure resources, and then build and deploy your
+BentoService as docker image to Azure Functions.
+
+.. code-block:: bash
+
+    $ bentoml azure-functions deploy azure-bentoml -b IrisClassifier:20200618124555_310CE0 --location westus
```

Comment:
the deployment name "azure-bentoml" is kinda confusing here

let's use something like `iris-classifier-demo`